docs/deployment/settings.md

```diff
@@ -296,9 +296,16 @@ kyuubi.frontend.protocols|THRIFT_BINARY|A comma separated list for all frontend
 kyuubi.frontend.proxy.http.client.ip.header|X-Real-IP|The http header to record the real client ip address. If your server is behind a load balancer or other proxy, the server will see this load balancer or proxy IP address as the client IP address, to get around this common issue, most load balancers or proxies offer the ability to record the real remote IP address in an HTTP header that will be added to the request for other devices to use. Note that, because the header value can be specified to any ip address, so it will not be used for authentication.|string|1.6.0
 kyuubi.frontend.rest.bind.host|&lt;undefined&gt;|Hostname or IP of the machine on which to run the REST frontend service.|string|1.4.0
 kyuubi.frontend.rest.bind.port|10099|Port of the machine on which to run the REST frontend service.|int|1.4.0
+kyuubi.frontend.ssl.exclude.protocols|SSLv2,SSLv3|SSL Versions to disable.|seq|1.7.0
```

Comment:
irrelevant with kyuubi.frontend.thrift.http.ssl.protocol.blacklist？

Reply:
```
  val FRONTEND_THRIFT_HTTP_SSL_PROTOCOL_BLACKLIST: ConfigEntry[Seq[String]] =
    buildConf("kyuubi.frontend.thrift.http.ssl.protocol.blacklist")
      .doc("SSL Versions to disable when using HTTP transport mode.")
      .version("1.6.0")
      .withAlternative("kyuubi.frontend.ssl.exclude.protocols")
      .stringConf
      .createWithDefault("SSLv2,SSLv3")
      .toSequence()
      .createWithDefault(Seq("SSLv2", "SSLv3"))
```
```
      .withAlternative("kyuubi.frontend.ssl.exclude.protocols")
```

Reply:
shall we add a alternative keys column in settings.md？

Reply:
`kyuubi.frontend.thrift.http.ssl.protocol.blacklist` is a legacy parameter added before.

and  pan suggested that using `exclude/include` instead of `blacklist/whitelist`.

Reply:
`exclude` is not specific to me. it's too ambiguous, is it `not supported(you shall not use it)`/`disabled(you can turn it on if you want)`/`not needed` or something else?

Reply:
How about
```
kyuubi.frontend.ssl.protocols.denied
kyuubi.frontend.ssl.protocols.allowed
```

Reply:
these sound like boolean parameters as they all end with a verb.

Reply:
does kyuubi.frontend.ssl mean it works for other frontend protocals?

Reply:
another thing confuses me and maybe confuse other users is the word protocol here, it can point to kyuubi frontend protocol or SSL version

Reply:
> does kyuubi.frontend.ssl mean it works for other frontend protocals?

Yes, the ssl parameters should be common for all the protocols